src/web/handlers.rs

```diff
@@ -65,7 +69,6 @@ pub async fn get_tiles(
     {
         tags.add_extra("country", &location.country());
         tags.add_extra("region", &location.region());
```

Comment:
Just curious, is it possible that `LocationResult::from_header(header, settings, &metrics)` could return incorrect values, like those reported by adM last week? If that's the case, we'd need to handle that so that no incorrect values get sent over.

Reply:
Yea, that's a separate issue #148 that I'll have a separate PR for